VTUL/dlp-access
# Enable adding / editing Collections through admin interface

**Enable adding / editing Collections through admin interface.**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-2395) (:star:)

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
Enables adding / editing Collections through admin interface.

# What's the changes? (:star:)
* Adds form that creates new Collection record
* Allows user to select existing Collection record to edit

# How should this be tested?
* Visit `/siteAdmin`, login and click on "New / Update Collection"
* Add identifier for Collection in the demo site and click "Confirm"
* Change a metadata field and click "Update Collection Metadata"
* Check that it changed the value in the sites Collection page (and please change it back)
*  Visit `/siteAdmin`, login and click on "New / Update Collection"
* Click "Create Collection"
* Fill out all required fields and submit
* Check that Collection was created in site
* (Probably would be a good idea to delete this test collection in dynamoDB)

# Additional Notes:
* branch: `LIBTD-2395`

# Interested parties
@yinlinchen 

(:star:) Required fields
